pages/server/development/aio.mdx

```diff
@@ -18,85 +20,110 @@ In that case, go to DBH server and run this command:
+            verified={false}
+          >
+            <DiscordEmbed slot="embeds" color="#00FF00" embed-title="Server Successfully Created">
+						  <DiscordEmbedDescription slot="description">
+							Click Here to Access Your Server
+							</DiscordEmbedDescription>
+							<DiscordEmbedFields slot="fields">
+                <DiscordEmbedField fieldTitle="Status:" inline inline-index="1">
+                  {' '}
+                  Created{' '}
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="User ID:" inline inline-index="2">
+                  {' '}
+                  126{' '}
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Type:" inline inline-index="3">
+                  {' '}
+                  aio{' '}
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Server Name:" inline inline-index="4">
+                  {' '}
+                  Untitled Server (settings -> server name){' '}
+                </DiscordEmbedField>
+              </DiscordEmbedFields>
+							<DiscordEmbedFooter
+                slot="footer"
+                footer-image="https://avatars.githubusercontent.com/u/66625200?v=4"
+                timestamp={new Date()}
+              >
+                Command Executed By: jonfirexbox (856176853719187506)
+              </DiscordEmbedFooter>
+            </DiscordEmbed>
+          </DiscordMessage>
+        </DiscordMessages>
 <br></br>
 For a donator server:
-<DiscordMessages>
-  <DiscordMessage
-    author="Jonfirexbox"
-    avatar="https://avatars.githubusercontent.com/u/66625200?v=4"
-    roleName="Donator"
-    roleColor="#ff00a0"
-    roleIcon="https://images-ext-1.discordapp.net/external/jcxosamsnMQwF-XU7dDsYt5Mz_ti0BRJc5pzpGSDPQ0/https/cdn.discordapp.com/role-icons/898041754564046869/576cbb697c65c8fffe1df17961401d8b.png?format=webp&quality=lossless"
-  >
-    DBH!server create-donator aio [optional server name]
-  </DiscordMessage>
-  <DiscordMessage
-    author="DanBot Hosting"
-    avatar="https://images-ext-1.discordapp.net/external/9j2_RacvCl8mly7hALccyH0PsM-tbJYm8twQKGf_3jY/https/cdn.discordapp.com/avatars/640161047671603205/718aa5783508125a02e3b244a1bf096e.png"
-    roleColor="#0329d3"
-    bot={true}
-    verified={false}
-  >
-    <DiscordEmbed slot="embeds" color="#00FF00">
-      <DiscordEmbedFields slot="fields">
-        <DiscordEmbedField fieldTitle="Status"> Created </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="Created for user ID">
-          {" "}
-          126{" "}
-        </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="Server name">
-          {" "}
-          Untitled Server (settings -> server name){" "}
-        </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="Type"> aio </DiscordEmbedField>
-        <DiscordEmbedField fieldTitle="WARNING">
-          {" "}
-          Please do not host game servers on java or AIO servers. If you need a
```

Comment:
This line is still needed for both donator and non donator.

Reply:
fixing now